keyboards/numatreus/keymaps/like_jis/keymap.c

```diff
@@ -0,0 +1,213 @@
+#include QMK_KEYBOARD_H
+#include "keymap_jp.h"
+
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+#endif
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+enum layer_number {
+  _BASE = 0,
+  _NUMPAD,
+  _LOWER,
+  _RAISE,
+  _ADJUST,
+};
+
+enum custom_keycodes {
+  BASE = SAFE_RANGE,
+  NUMPAD,
+  KANJI,
+  LOWER,
+  RAISE,
+  ADJUST,
+  RGBRST
+};
+
+enum tapdances{
+  TD_CODO = 0,
+  // TD_MNUB,
+};
+
+// Layer Mode aliases
+#define KC_LOWER LOWER
+#define KC_RAISE RAISE
+#define KC_ADJST ADJUST
+#define KC_DLNP NUMPAD //DF(_NUMPAD)
+#define KC_DLBS BASE //DF(_BASE)
+
+#define KC______ KC_TRNS
+#define KC_XXXXX KC_NO
```

Comment:
These should be replaced with `_______` and `XXXXXXX` (7 characters) in the keymap.

Reply:
It has changed.